packages/react/src/PageHeader/PageHeader.module.css

```diff
@@ -0,0 +1,220 @@
+:root {
+  /* Grid Row Order */
+  --grid-row-order-context-area: 1;
+  --grid-row-order-leading-action: 2;
+  --grid-row-order-breadcrumbs: 2;
+  --grid-row-order-title-area: 2;
+  --grid-row-order-trailing-action: 2;
+  --grid-row-order-actions: 2;
+  --grid-row-order-description: 3;
+  --grid-row-order-navigation: 4;
+
+  /* Title Area Region Order */
+  --title-area-region-order-leading-visual: 0;
+  --title-area-region-order-title: 1;
+  --title-area-region-order-trailing-visual: 2;
+
+  /* Context Area Region Order */
+  --context-area-region-order-parent-link: 0;
+  --context-area-region-order-context-bar: 1;
+  --context-area-region-order-context-area-actions: 2;
+}
```

Comment:
Something that crossed my mind reviewing this, what are our feelings about putting these css variables in the `:root`? I don't believe the css modules conversion does anything about it and it ends up in `:root` in prod. @langermank

Reply:
I agree, it might be better to tuck these into the `.PageHeader` class instead of `:root`